lms/templates/dashboard.html

```diff
@@ -207,8 +207,12 @@
                 show_consent_link = (session_id in consent_required_courses)
                 course_overview = enrollment.course_overview
                 resume_button_url = resume_button_urls[dashboard_index]
+
+                enrollment_date = None
+                if course_overview.self_paced:
+                    enrollment_date = enrollment.created
```

Comment:
Are you able to guarantee `enrollment` exists here too? Otherwise you might run into a `None.created`

Reply:
Good call